docker/sensorflux-backend/README.md

```diff
@@ -0,0 +1,66 @@
+# Sensorflux backend compose
+
+Sample backend based on InfluxDB + Grafana + Chronograf.
+
+Note: mainly based on https://github.com/jkehres/docker-compose-influxdb-grafana.git . Full documentation of the underlying setup can be found there.
+
+## Quick Start
+
+To start the app:
+
+1. Install [docker-compose](https://docs.docker.com/compose/install/) on the docker host.
+1. Clone this repo on the docker host.
+1. Optionally, change the default credentials or Grafana provisioning.
+1. Run the following command from the root of the cloned repo:
```

Comment:
i would maybe change "the root of the cloned repo" on line 14, to "the sensorflux-backend directory" / "this directory". As the command only works from this directory, it might be confusing to people who don't know docker and only cloned sensorflux.